bundles/ch.elexis.core.ui/src/com/tiff/common/ui/datepicker/DatePicker.java

```diff
@@ -255,14 +255,36 @@ private void onKeyDown(KeyEvent e) {
 
 		private void onMouseDown(MouseEvent e) {
 			int day = getDayFromPoint(e.x, e.y);
-
-			if (day > 0) {
+		
+			Listener[] verifyListeners = getVerifyListeners();
+			boolean doIt = true;
+			
+			if (verifyListeners != null) {
+				for (int i = 0; i < verifyListeners.length && doIt; ++i ) {
+					doIt = verifyEnteredDay(day, verifyListeners[i]);
```

Comment:
Das `doIt` als Abbruchbedingung innerhalb der Schleife gefällt mir nicht.
Bitte explizite prüfung in nächster Zeile, aber nicht als Teil der schelfei!